cmd/user-cluster-webhook/main.go

```diff
@@ -77,9 +77,7 @@ func main() {
 		log.Fatalw("Failed to get user cluster kubeconfig")
 	}
 
-	// /////////////////////////////////////////
-	// create manager
-
+	// This case unusual in a sense that we need to configure and start managers for both user and seed clusters.
```

Comment:
This is missing an "is" ("This case IS unusual") and also, it's generally not _that_ unusual. Our master-ctrl-mgr does the same for master/seed clusters, even dynamically managing Managers at runtime (whenever new Seed objects appear or are updated). So this all is not such a crazy thing as you might think it is 😁